tests/mbedtls_test/Makefile

```diff
@@ -0,0 +1,42 @@
+APPLICATION= lwip_mbedtls_test
+
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD ?= native
+
+USEPKG += mbedtls
+USEMODULE += mbedtls_entropy
+USEMODULE += ipv4_addr
+USEMODULE += lwip_arp
+USEMODULE += lwip_ipv4
+USEMODULE += lwip_ipv6
+USEMODULE += lwip_dhcp_auto
+USEMODULE += lwip lwip_netdev
+USEMODULE += lwip_udp
+USEMODULE += lwip_tcp
+USEMODULE += sock_tcp
+
+# Add also the shell, some shell commands
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+# Add atwinc15x0 driver
+USEMODULE += atwinc15x0
+# tinydtls needs crypto secure PRNG
+USEMODULE += prng_sha1prng
+USEMODULE += random
+
+CFLAGS += -DWIFI_SSID="\"ADD-WIFI-SIDI\"" -DWIFI_PASS="\"ADD-PASSWORD\""
+CFLAGS += -DATWINC15X0_PARAM_SSN_PIN="GPIO_PIN(1, 28)" -DATWINC15X0_PARAM_RESET_PIN="GPIO_PIN(0, 6)" \
+	-DATWINC15X0_PARAM_IRQ_PIN="GPIO_PIN(1, 7)" -DATWINC15X0_PARAM_CHIP_EN_PIN="GPIO_PIN(0, 27)" \
+	-DATWINC15X0_PARAM_WAKE_PIN="GPIO_PIN(0, 7)"
+CFLAGS += -DCONFIG_ENTROPY_SOURCE_ADC_HMIN="200"
+CFLAGS += -DISR_STACK_SIZE=4096
+CFLAGS += -DHOST_SERVER="\"10.0.110.117\""
```

Comment:
Let's set this using a shell command. Then we also don't need a separate thread - just bump `THREAD_STACKSIZE_MAIN`  if necessary. 

```C
static int cmd_tls_connect(int argc, char **argv)
{
    if (argc < 3) {
        printf("usage: %s <host> <port> [data]\n", argv[0]);
        return -1;
    }

    return _lwip_mbedtls_client(argv[1], argv[2]);
}

static const shell_command_t shell_commands[] = {
    { "connect", "Perform a TLS connection", cmd_tls_connect },
    { NULL, NULL, NULL }
};
```